src/app.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from "react";
+import Navigation from './components/navigation.js';
+import Processors from './pages/processors.js';
+import Videocards from './pages/videocards.js';
+import Mainboards from './pages/mainboards.js';
+import Memories from './pages/memories.js';
+import Devices from './pages/devices.js';
+import './styles/app.css';
+
+class App extends React.Component {
+    render() {
+        return (
+            <Navigation>
+                <Processors path="processors" />
```

Comment:
In you case you render pages inside navigation
I think you want ot render it like this

```
<>
            <Navigation/>
            <Videocards path="videocards" />
            <Mainboards path="mainboards" />
            <Memories path="memories" />
            <Devices path="devices" />
</>
```

https://reactjs.org/docs/fragments.html